### PR TITLE
Changed the content on landing page about python dependency

### LINKED ISSUE #1046: Update references to python 3.5
### Where is the problem?

Quick Start section on https://qiskit.org/

https://qiskit.org/documentation/install.html

### What is the problem?

Installation instructions list python 3.5+ as a requirement for Qiskit. Qiskit 0.22.0 is the last version to support python 3.5, so we should recommend python 3.6+.

### What would you expect?

Python 3.6+

### PR DESCRIPTION
**Pull request by aryansi225**
Fixes #1046 

Changed the components/landing/TheQuickStart/StartLocally.vue file to reflect the correct version of Python required for qiskit.

### CODE REVIEW

**Review by korgan00 — APPROVED** [review]
Thank you for your collaboration.

**Review by eddybrando — APPROVED** [review]
Nice, thank you! 👍

**Comment by aryansi225** [comment]
Can someone having write access merge this PR, provided you find this PR fit to be merged.

**Comment by aryansi225** [comment]
@y4izus Thanks for merging.

**Pull request merged**
